tests/unit/kvmagent/test_volume_cache_cancel.py

```
@@ -0,0 +1,380 @@
+import platform
+import signal
+import sys
+import types
+import unittest
+from unittest.mock import patch
+
+
+platform.freedesktop_os_release = lambda: {"ID": "centos", "VERSION_ID": "7"}
+platform.machine = lambda: "x86_64"
+
+os.makedirs("/tmp/zstack-test-log", exist_ok=True)
```

Ruff (0.15.20)

[error] Probable insecure usage of temporary file or directory: "/tmp/zstack-test-log"

(S108)